instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -146,3 +140,31 @@ def test_simpletheme_optout(self):
             self.assertNotIn('SIMPLETHEME_ENABLE_DEPLOY', parsed_vars)
             self.assertNotIn('SIMPLETHEME_SASS_OVERRIDES', parsed_vars)
             self.assertNotIn('EDXAPP_DEFAULT_SITE_THEME', parsed_vars)
+
+    def test_get_contrasting_font_color(self):
+        """
+        Tests if the automatic font color selection is working properly
+        """
+        # Create objects
+        OpenEdXInstanceFactory(name='Integration - test_colors_applied', deploy_simpletheme=True)
+        instance = OpenEdXInstance.objects.get()
+
+        # Test if the font color is correctly returned depending on the background color
+        test_colors = [
```

Comment:
@giovannicimolin, use `ddt.data` for this.

Reply:
:+1: